cmd/timescale-prometheus/main.go

```diff
@@ -198,6 +204,14 @@ func main() {
 	}
 
 	appVersion := pgmodel.VersionInfo{Version: version.Version, CommitHash: version.CommitHash}
+	isSafe, err := isDBVersionSafe(&cfg.pgmodelCfg)
+	if err != nil {
+		log.Error("err", errors.Wrap(err, "db-version safe check"))
```

Comment:
@cevian are we using `errors` or are we migrating to `fmt.Errorf("%w")`?

Reply:
IMHO errors should be better since we want to approach a modular way. This makes it much easy to propagate down the error stack to the cause.

Reply:
we made the decision to use `fmt.Errorf("%w")` in this project because it is becoming the standard